components/brave_wallet_ui/components/desktop/card-headers/nft-asset-header.tsx

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+
+// utils
+import Amount from '../../../utils/amount'
+import { getLocale } from '../../../../common/locale'
+
+// Styled Components
+import {
+  CircleButton,
+  HeaderTitle,
+  ButtonIcon,
+  SendButton
+} from './shared-card-headers.style'
+import { Row } from '../../shared/style'
+
+interface Props {
+  assetName?: string
+  tokenId?: string
+  onBack: () => void
+}
+
+export const NftAssetHeader = ({ assetName, tokenId, onBack }: Props) => {
+  return (
+    <Row
+      padding='26px 0px'
+      justifyContent='space-between'
+      alignItems='center'
+    >
+      <Row
+        justifyContent='flex-start'
+      >
+        <CircleButton
+          size={28}
+          marginRight={16}
+          onClick={onBack}
+        >
+          <ButtonIcon
+            size={16}
+            name='arrow-left'
+          />
+        </CircleButton>
+        <HeaderTitle>
+          {assetName}&nbsp;{tokenId ? `#${new Amount(tokenId).toNumber()}` : ''}
+        </HeaderTitle>
+      </Row>
+      <SendButton>{getLocale('braveWalletSend')}</SendButton>
```

Comment:
Should this `SendButton` route to `brave://wallet/send`?

Reply:
Fixed here: 8cec20024b47506f0b3efb3edf2b2fa09aadc62e